pai-management/quick-start/services-configuration.yaml.template

```diff
@@ -43,7 +43,8 @@ cluster:
     #docker-username: <username>
     #docker-password: <password>
 
-    docker-tag: latest
+    # Should match the PAI version, pai-0.6.y for the PAI release 0.6.
+    docker-tag: pai-0.6.y
```

Comment:
>pai-0.6.y [](start = 16, length = 9)

I suggest you put a specific version in comment. and leave the default value to latest. people may not care about specific version.

Reply:
In my opinion, user should always use a latest stable release.
The latest code/image may break.

Reply:
that I agree. Maybe we should always release the latest stable docker image that allows user to use the latest tag,  and push a pre-release docker images somewhere else, instead of the "lastest"?

---
In reply to: [198000722](https://github.com/Microsoft/pai/pull/769#discussion_r198000722) [](ancestors = 198000722)

Reply:
It works, but maybe a bit weird.
When 'latest' is not always the latest, what does it mean?

And should we encourage user to use the edge version?
When we recommend the usage of 'latest', we are encouraging them to use the newest feature.
When we set the specific 'version' as default, we are encouraging them to use stable version.

Reply:
got it. make sense.

---
In reply to: [198007885](https://github.com/Microsoft/pai/pull/769#discussion_r198007885) [](ancestors = 198007885)

Reply:
Yeah, good idea.

Reply:
I suggest we don't put any release version dependency in the code. We can use 'latest' and add some comments describing how to specify the version in the yaml file.

Reply:
I see, what about don't merge this into master branch?
Master branch should always use the 'latest', but this branch is for 0.6, it's reasonable to lock it to specific version.